src/coreclr/build.cmd

```diff
@@ -1010,9 +1021,12 @@ REM === Helper routines
 REM ===
 REM =========================================================================================
 
-:CrossgenFailure
+:ExitWithError
 exit /b 1
 
+:ExitWithCode
+exit /b !__exitCode!
```

Comment:
Actually, maybe document the weird subroutine behavior here, so someone doesn't try to remove it in the future thinking it's just someone trying to do "single return". (And if someone hits a similar issue, maybe they'll see this and get their fix without as much headache.)

Reply:
Sounds good.

Reply:
Is the expectation of this method that a zero exit code is acceptable? Basically is this explicitly an error routine or a general purpose exit routine?

Reply:
I think a general purpose exit routine is acceptable. That's why there's also ExitWithError which exits with 1.